src/Download.java

```diff
@@ -78,11 +78,12 @@ static void main(boolean dryRun, String... args) {
       // Emit warning when using an archived JDK build
       if (website.isArchivedUri(uri)) {
         GitHub.warn(
-            "JDK resolved to an archived build!\n"
-                + "These older versions of the JDK are provided\n"
-                + " to help developers debug issues in older systems.\n"
-                + "They are not updated with the latest security patches\n"
-                + " and are not recommended for use in production.");
+                """
```

Comment:
3 lines, one per sentence, would be nicer.

```java
"""
JDK resolved to an archived build!
These older versions of the JDK are provided to help developers debug issues in older systems.
They are not updated with the latest security patches and are not recommended for use in production.
"""
```

Reply:
fixed